src/digitalmarketplace/components/banner/template.njk

```diff
@@ -0,0 +1,17 @@
+<div
```

Comment:
GOV.UK Publishing Components has their banner as a `<section>` rather than a generic `<div>`; is there a reason to prefer a `<div>` over a `<section>` for our banner?